integration_tests/test_c_interop_03.py

```diff
@@ -20,7 +20,8 @@ def test_c_callbacks():
     xi32: i32
     xi32 = 3
     p: CPtr
-    p_c_pointer(xi32, p)
+    p_c_pointer(pointer(xi32), p)
+    print(pointer(xi32), p)
```

Comment:
Its output verifies whether the addresses were correctly stored. Having these prints helps a lot to verify our implementations.